UserBundle/Document/Group.php

```diff
@@ -7,7 +7,7 @@
 
 /**
  * @ODM\Document(
- *  collection="group_document",
+ *  collection="group",
```

Comment:
this should not be done, if you do it, you cannot run db.group.find() in the mongo shell